es/03_5_Entendiendo_El_Descriptor.md

```diff
@@ -0,0 +1,132 @@
+# 3.5: Comprensión del descriptor
+
+> :information_source: **NOTA:** Esta sección se ha agregado recientemente al curso y es un borrador inicial que aún puede estar pendiente de revisión. Lectura con precaución.
```

Comment:
`Lectura con precaución.` --> `Precaucion lector.`?

Reply:
Actually I used to traduce this as "Lectura con advertencias"

Reply:
Sure. You noted me that in the translation of `A3`, and I accepted. Keep it as it is.